salt/modules/file.py

```diff
@@ -2494,6 +2496,17 @@ def blockreplace(path,
         If markers are not found and set to ``True`` then, the markers and
         content will be prepended to the file.
 
+    insert_before_match
+        If markers are not found, this parameter can be set to a regex which will
+        insert the block before the first found occurrence in the file.
+
+        .. versionadded:: Fluorine
+
+    insert_after_match
+        If markers are not found, this parameter can be set to a regex which will
+        insert the block after the first found occurrence in the file.
+
+        .. versionadded:: Fluorine
```

Comment:
We have already passed our feature freeze for Fluorine, so these `versionadded` items will need to be updated to `Neon`.